spec/helpers/spec-helper.js

```diff
@@ -3,6 +3,11 @@ const assert = require('assert');
 
 
 global.expect = function(e) {
+    
+    function expected(e, message, a) {
```

Comment:
I don't see this used anywhere below

Reply:
It's a convenience function which is only used in the same file.
e.g. https://github.com/stefnotch/gl-matrix/blob/master/spec/helpers/spec-helper.js#L86